x-pack/plugins/code/public/components/file_tree/file_tree.tsx

```diff
@@ -123,90 +110,85 @@ export class CodeFileTree extends React.Component<Props> {
               onClick={onClick}
             >
               {forceOpen ? (
-                <EuiIcon type="arrowDown" size="s" className="codeFileTree--icon" />
+                <EuiIcon type="arrowDown" size="s" className="codeFileTree__icon" />
               ) : (
-                <EuiIcon type="arrowRight" size="s" className="codeFileTree--icon" />
+                <EuiIcon type="arrowRight" size="s" className="codeFileTree__icon" />
               )}
-              <EuiIcon
-                type={forceOpen ? 'folderOpen' : 'folderClosed'}
-                data-test-subj={`codeFileTreeNode-Directory-Icon-${node.path}-${
```

Comment:
@daveyholler Hey Davey, you missed this attribute in your new change. This leads to functional test failures in the CI. Other than this, everything is good now.